Packs/ThreatX/Integrations/ThreatX/README.md

```diff
@@ -69,6 +69,11 @@
 <td style="width: 511px;">IP address or CIDR, for example: "10.1.1.1" or "10.1.1.0/24".</td>
 <td style="width: 83px;">Required</td>
 </tr>
+<tr>
+<td style="width: 146px;">description</td>
+<td style="width: 511px;">The description of the IP record in the block list.</td>
```

Comment:
please add the default value to the 3 added args

Reply:
done